src/controller/userController.ts

```diff
@@ -9,10 +9,13 @@ export async function login(req: Request, res: Response) {
 
     try {
         const data = await selectUser(userData);
+        if (data?.err) {
+            throw new Error(data.err);
```

Comment:
When an error is thrown in the controller component, the whole instance of node will be stopped.